packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -202,6 +208,24 @@ describe('logs', () => {
 
       resetExperimentalFeatures()
     })
+
+    it('should send reports', () => {
+      if (!isChromium()) {
+        pending('no ReportingObserver support')
+      }
+      const logger = new Logger(noop)
+      const logErrorSpy = spyOn(logger, 'log')
+      const reportingObserverStub = stubReportingObserver()
+      updateExperimentalFeatures(['forward-reports'])
+      originalStartLogs({ ...baseConfiguration, forwardReports: ['intervention'] }, logger)
```

Comment:
**💬 suggestion:**  it could be interesting to add a test for the expected default behavior as well‏

Reply:
By the expected default behaviour, you mean without the FF?

Reply:
Okay, done